judgments/forms/search_forms.py

```diff
@@ -35,6 +35,24 @@ def _get_choices_by_group(courts):
 TRIBUNAL_CHOICES = _get_choices_by_group(all_courts.get_grouped_selectable_tribunals())
 
 
+def get_short_identifiers(top_level_choices):
+    all_shorts = set()
+    for key, value in top_level_choices:
+        if isinstance(value, list):
+            all_shorts = all_shorts | set(x[0].partition("/")[0] for x in value)
+    return all_shorts
+
+
+class CourtTribunalField(forms.MultipleChoiceField):
+    def validate(self, *args, **kwargs):
+        short_identifiers = get_short_identifiers(self.choices)
+        try:
+            return super().validate(*args, **kwargs)
+        except ValidationError as e:
+            if e.params.get("value") in short_identifiers:
+                return  # acceptable value
```

Comment:
Is it okay to return nothing here? Should we explicitly return `None` or whatever this expects us to return?

Reply:
Sure, this is purely stylistic, and explicit is better than implicit.